tests/test_client_ws.py

```diff
@@ -84,6 +174,7 @@ async def test_ws_connect_with_params(ws_key: Any, loop: Any, key_data: Any) ->
         hdrs.SEC_WEBSOCKET_ACCEPT: ws_key,
         hdrs.SEC_WEBSOCKET_PROTOCOL: "chat",
     }
+    resp.connection.protocol.read_timeout = None
```

Comment:
This seems like a breaking change if this now needs to be set in the below tests

Reply:
I don't think so. This is because `Mock` is not comparable with `int` for the purposes of `max`.